pkg/controller/common/service_control_test.go

```diff
@@ -285,7 +288,7 @@ func Test_applyServerSideValues(t *testing.T) {
 			}},
 		},
 		{
-			name: "Reconciled ClusterIP[s] is not used if the reconciled ClusterIP[s] are not valid IPs",
+			name: "Reconciled ClusterIP[s] is also used if the reconciled ClusterIP[s] are not valid IPs",
```

Comment:
Not sure what we are testing now with this case. It's equivalent to the previous case 'Reconciled ClusterIP/ClusterIPs/Type/SessionAffinity is used if expected ClusterIP/Type/SessionAffinity is empty'.